rero_ils/modules/patrons/templates/rero_ils/_patron_profile_history.html

```diff
@@ -21,7 +21,7 @@
     <div class="row p-2 bg-dark rounded text-light">
       <div class="col-lg-6">{{ _('Title') }}</div>
       <div class="col-lg-4">{{ _('Author(s)') }}</div>
-      <div class="col-lg-2">{{ _('Return date') }}</div>
+      <div class="col-lg-2">{{ _('Check-in date') }}</div>
```

Comment:
When a document is returned to the library, we use the term *checkin*. Do you think it's wrong?